test/features/ui/widgets/table_test.dart

```diff
@@ -0,0 +1,257 @@
+import 'package:flyde/core/console/text_alignment.dart';
+import 'package:flyde/features/ui/render/widget.dart';
+import 'package:flyde/features/ui/widgets/table.dart';
+import 'package:test/test.dart';
```

Comment:
Can we stick with the order:
```
External

lib (with package: because we are in the tests dir)

test (relative)
```

Reply:
```
dart 

external (e.g. test)

lib

relative imports
```

I don't want to mix `dart:` imports with `package:` imports

Reply:
I resolve the other related comments to keep the review a little cleaner

Reply:
Jup, simply forgot about the dart imports, thats how I do it already 👍🏻